4_advanced_usage/75_separate_database.txt

```diff
@@ -6,6 +6,84 @@ The setup is done in two places: on the database server and on the Rudder root s
 
 It also allows moving an existing database to another server.
 
+[TIP]
+
+.Use different user and database names
+
+====
+
+It can be useful, for example if you want to share you database server between several Rudder root servers (see note below),
+to use a different database for your Rudder root server. To do so:
+
+* Create the new database (replace `alternate_user_name`, `alternate_base_name` and specify a password):
+
+----
+
+su - postgres -c "psql -q -c \"CREATE USER alternate_user_name WITH PASSWORD 'GENERATE_A_PASSWORD'\""
+su - postgres -c "psql -q -c \"CREATE DATABASE alternate_base_name WITH OWNER = alternate_user_name\""
+
+----
+
+* Initialize it. First copy the initialization script:
+
+----
+
+ cp /opt/rudder/etc/postgresql/reportsSchema.sql /opt/rudder/etc/postgresql/reportsSchema-alternate.sql
+
+----
+
+* In the copied file, change the:
+
+----
+
+ALTER database rudder SET standard_conforming_strings=true;
+
+----
+
+To:
+
+----
+
+ALTER database alternate_base_name SET standard_conforming_strings=true;
+
+----
+
+* Then apply the script:
+
+----
+
+su - postgres -c "psql -q -U alternate_user_name -h localhost -d alternate_base_name \
+     -f /opt/rudder/etc/postgresql/reportsSchema-alternate.sql"
+
+----
+
+* Follow the normal instructions, with two differences :
```

Comment:
What normal instructions? Can you clarify, as in "above" or with a link?

BTW, "normal" is a word best avoided. I propose "standard" instead.